### PR TITLE
[Interop branch] Log improvements - part 1

### PR DESCRIPTION
**Pull request by mratsim**
Part 1 of log improvements (#437) so that we don't drawn other implementers teams in messages
Target is interop branch.

The main changes are:
- the service/category/process, this will make it easier to filter the logs in the future. Everything should be rediscussed so that we have something that makes sense and easy to navigate through.
  For example maybe we should use trigrams for service code as we're really short on characters per line
- AFAIK chronicles priority is trace/debug/info/notice/warn/error:
  - Some logs were deleted as they were duplicate of logs given by a procedure called just after i.e. "Slot start"/"Scheduling slot action"
  - some were very noisy "Fork chosen"/"Preparing for fork choice"/"Updated head", only the result was kept.
- A couple of logs relegated to tracing (growing attestation pool, ignoring subset attestation, checking attestations)

No logic was changed in this PR, future changes:

1. skip block received/attestation received for our own block/attestation.
2. try to make most non-warning messages fit on a single line of a laptop screen.

### CODE REVIEW

File: beacon_chain/attestation_pool.nim [hunk 1]
```diff
@@ -93,14 +93,20 @@ proc slotIndex(
...
+      attestationSlot =  $shortLog(attestationSlot),
+      service = "attestation_pool",
+      category = "init",
+      process = "first_attestation"
     pool.startingSlot =
       state.finalized_checkpoint.epoch.compute_start_slot_of_epoch()
 
   if pool.startingSlot + pool.slots.len.uint64 <= attestationSlot:
-    debug "Growing attestation pool",
+    trace "[Attestation Pool] Growing attestation pool",
```
Comment (zah) [inline beacon_chain/attestation_pool.nim]: if we want to have [Subsystem Labels] like these, I can teach Chronicles how to produce them.

Comment (arnetheduck) [reply beacon_chain/attestation_pool.nim]: isn't that what `topic` is for, in chronicles already? that seems superior..

File: beacon_chain/attestation_pool.nim [hunk 2]
```diff
@@ -93,14 +93,20 @@ proc slotIndex(
     # to start counting at the last finalized epoch start slot - anything
     # earlier than that is thrown out by the above check
     info "First attestation!",
-      attestationSlot =  $shortLog(attestationSlot)
+      attestationSlot =  $shortLog(attestationSlot),
+      service = "attestation_pool",
```
Comment (arnetheduck) [inline beacon_chain/attestation_pool.nim]: there's `logScope` for this kind of stuff as well

**Comment by mratsim** [comment]
Screenshots  on a 13" Mac / Retina:




The `topics =` will be made optional in chronicles, and hopefully we can also get a way to automatically get the calling function name in Nim (https://github.com/nim-lang/Nim/issues/8212)

**Pull request merged**